man/rpm-ostree.xml

```diff
@@ -493,8 +493,13 @@ Boston, MA 02111-1307, USA.
 
           <para>
             <command>replace</command> to replace base packages.
-            Currently, only local RPM replacements are supported:
-            one must directly provide the RPMs to substitute in.
+            Local RPM replacements are supported: one must directly 
+            provide the RPMs to substitute in. Also supports directly 
```

Comment:
Nit: the phrasing feels odd here but I don't have a better suggestion.

Reply:
What about something like: `Currently 3 options are supported: Local RPM files and pulling builds from Bodhi or Koji which are Fedora build/test tools.`

Reply:
Used that thanks @cgwalters